docs_app/content/navigation.json

```diff
@@ -24,7 +24,13 @@
 		{
 			"url": "guide/overview",
 			"title": "Overview",
-			"tooltip": "RxJS Overview"
+			"tooltip": "RxJS Overview",
+			"children": [
+				{
+					"url": "guide/subject",
+					"title": "Subjects in RxJS"
```

Comment:
'Subjects' would be totally fine for the title. I think it is totally clear that it is about rxjs :D

Reply:
I was just trying to clarify that this was "Subjects, the RxJS construct" vs. "Subjects (e.g. topics) about RxJS)," mostly for newer readers.  I can just change it to "Subjects" though -- that may be clear enough.